compiler/resolution/visibleCandidates.cpp

```diff
@@ -28,33 +28,35 @@
 #include "stringutil.h"
 #include "symbol.h"
 
-static void doGatherCandidates(CallInfo&                  info,
-                               Vec<FnSymbol*>&            visibleFns,
-                               bool                       compilerGenerated,
-                               Vec<ResolutionCandidate*>& candidates);
+static void       doGatherCandidates(CallInfo&                  info,
+                                     Vec<FnSymbol*>&            visibleFns,
+                                     bool                       generated,
+                                     Vec<ResolutionCandidate*>& candidates);
```

Comment:
One day you must tell me how you do this without inserting all the spaces by hand